wallet/how-to/connect/index.md

```diff
@@ -0,0 +1,210 @@
+---
+description: Connect to MetaMask via EIP-6963.
+sidebar_position: 1
+---
+
+# Connect to MetaMask
+
+You can connect your dapp to users' MetaMask wallets by detecting MetaMask in their browsers and
+connecting to their accounts.
+
+This page describes how to connect to MetaMask using the wallet detection mechanism introduced by
+[EIP-6963](https://eips.ethereum.org/EIPS/eip-6963).
+EIP-6963 enables [wallet interoperability](../../concepts/wallet-interoperability.md), and shifts
+dapps from relying solely on [`window.ethereum`](detect-metamask.md) for wallet detection.
+If a user has multiple wallet browser extensions installed, you can detect multiple wallets and
+connect to each one without conflict.
+
+You can connect to MetaMask [using third-party libraries](#use-third-party-libraries) or
+[directly using the Wallet API](#connect-to-metamask-directly).
+
+## Use third-party libraries
+
+You can connect to MetaMask using the following third-party libraries that support EIP-6963:
+
+- [Wagmi 2+](https://wagmi.sh)
+- [Web3Modal 3+](https://docs.walletconnect.com/web3modal/about)
+- [MIPD Store](https://github.com/wevm/mipd)
+- [RainbowKit](https://www.rainbowkit.com/)
+
+## Connect to MetaMask directly
```

Comment:
Should this be something like, "Connect using the Wallet API"?

Reply:
The Wallet API isn't explicitly mentioned in most of our other how-tos, so I hesitate to add it to the header here, for consistency. We're really just contrasting that you're implementing it directly in your dapp code vs. via 3rd-party.